schemas/post-artifact-request.yml

```diff
@@ -31,11 +31,35 @@ oneOf:
            always using `application/json` for JSON artifacts.
         type:       string
         maxLength:  255
+      contentMD5:
+        description: |
+          Base64 encoded MD5 sum of the artifact. If specified, it **must**
+          also be given as value to the `Content-MD5` header when uploading
+          to S3.
+          **Warning**, this will likely be required in the future, and **must**
+          be specified if `contentSHA256` is specified.
+        type:       string
```

Comment:
If the queue returns the headers as per your proposal in https://bugzilla.mozilla.org/show_bug.cgi?id=1190016#c5 section C) then I think it would be nice for the `createArtifact` (or `defineArtifact` if we decide to rename it) to receive both the hashes in hex for the sake of conventionality (e.g. command line tools typically report in hex, so it is the one people are most familiar with when troubleshooting/double checking the hash, etc).